doc/api/training/smd_model_parallel.rst

```diff
@@ -1,30 +1,30 @@
 Distributed model parallel
 --------------------------
 
-Amazon SageMaker Distributed Model Parallel (SMP) is a model parallelism library for training
+The Amazon SageMaker distributed model parallel library is a model parallelism library for training
 large deep learning models that were previously difficult to train due to GPU memory limitations.
-SMP automatically and efficiently splits a model across multiple GPUs and instances and coordinates model training,
+The library automatically and efficiently splits a model across multiple GPUs and instances and coordinates model training,
 allowing you to increase prediction accuracy by creating larger models with more parameters.
 
-You can use SMP to automatically partition your existing TensorFlow and PyTorch workloads
-across multiple GPUs with minimal code changes. The SMP API can be accessed through the Amazon SageMaker SDK.
+You can use the library to automatically partition your existing TensorFlow and PyTorch workloads
+across multiple GPUs with minimal code changes. The library's API can be accessed through the Amazon SageMaker SDK.
 
-Use the following sections to learn more about the model parallelism and the SMP library.
+Use the following sections to learn more about the model parallelism and the library.
 
 .. important::
-   SMP only supports training jobs using CUDA 11. When you define a PyTorch or TensorFlow
+   The model parallel library only supports training jobs using CUDA 11. When you define a PyTorch or TensorFlow
    ``Estimator`` with ``modelparallel`` parameter ``enabled`` set to ``True``,
    it uses CUDA 11. When you extend or customize your own training image
    you must use a CUDA 11 base image. See
-   `Extend or Adapt A Docker Container that Contains SMP
+   `Extend or Adapt A Docker Container that Contains the Model Parallel Library
```

Comment:
should the first letter be capital or small ?

Reply:
It's a link to a title, so it's capped.